Source/AtomicEditor/Application/AEEditorApp.cpp

```diff
@@ -158,9 +158,10 @@ void AEEditorApp::HandleJSError(StringHash eventType, VariantMap& eventData)
     String errFilename = eventData[P_ERRORFILENAME].GetString();
     //String errStack = eventData[P_ERRORSTACK].GetString();
     int errLineNumber = eventData[P_ERRORLINENUMBER].GetInt();
+    int errRealLineNumber = vm_->GetRealLineNumber("AtomicEditor/out/" + errFilename, errLineNumber);
```

Comment:
Hm, check that the error happened in a source mapped file, somehow?  How does this handle normal JS errors in non-sourcemapped files?